scripts/xtalk_dayenu_baselines.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+# Copyright 2019 the HERA Project
+# Licensed under the MIT License
+
+"Command-line driver script for xtalk Filtering with DAYENU that allows for parallelization across baselines."
```

Comment:
Say a bit more about how that parallelization works. What's loaded and filtered and what's written to disk?

Reply:
I've added more explicit documentation.